.circleci/config.yml

```diff
@@ -47,6 +47,16 @@ jobs:
       - run: mkdir -p ~/.ssh
       - run: ssh-keyscan -H github.com >> ~/.ssh/known_hosts
       - run: bash .circleci/commit.sh
+  scripts:
+    docker:
+      - image: circleci/node:16
+    working_directory: ~/repo
+    steps:
+      - attach_workspace:
+          at: ~/repo
+
+      # Run scripts to confirm they still compile and execute without error
+      - run: yarn run-script scripts/rpc/getSaleHistory.ts --network mainnet
```

Comment:
i see the motivation just not sure we need scripts to be verified on every CI run. your call :)

Reply:
we need something otherwise it'll regress and go unnoticed for awhile. Just like the deploy script this morning - burns time when we finally do want to run it again. And something like this we won't revisit often, this is just something to help keep it alive.